localdev/docker-compose.yml

```diff
@@ -14,6 +14,8 @@ services:
       JENKINS_SSH_USER_NAME: jenkins
       JENKINS_DOCKER_USER_NAME: docker_user
       JENKINS_DOCKER_USER_PASS: docker_pass
+      SLACK_TEAM_DOMAIN: "lovelysystems"
+      SLACK_TOKEN: "valid_slack_token"
```

Comment:
is there a reason why you didn't add all three env variables?

Reply:
BASE URL NOT REQUIRED when using team domain and token